### PR TITLE
Display Slack webhook fields in stage edit form

### PR DESCRIPTION
**Pull request by michael-bouvy**
Related to #837 

image

I'm not quite sure that the tests location is correct (tightly coupled), but I couldn't figure where else to place it. Please tell me if it's the wrong place.

/cc @zendesk/samson

### Tasks
 - [ ] :+1: from team

### CODE REVIEW

File: plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <legend>Slack</legend>
 
   <% stage = form.object %>
-  <% stage.slack_webhooks.build unless stage.slack_webhooks.last.try(:webhook_url).blank? %>
+  <% stage.slack_webhooks.build unless stage.slack_webhooks.size > 0 && stage.slack_webhooks.last.webhook_url.blank? %>
```
Comment (grosser) [inline plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb]: what does this change do ? ... now it does 2 queries instead of 1 :(

Comment (grosser) [inline plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb]: ahhhh it handles missing hook and empty hook differently ... how about:

`unless (last = stage.slack_webhooks.last) && last.webhook_url.blank?`

**Comment by grosser** [comment]
great to see a test :D

File: plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb (hunk 1 above)
Comment (michael-bouvy) [inline plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb]: Works, thanks for the tip! 😉 

How about removing the condition anyways? As previously said in the issue, I couldn't reproduce missing hook URL.

Comment (grosser) [inline plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb]: this is meant to not add a blank hook when a update/create failed, so not super important ...

**Pull request merged**